### PR TITLE
Fix the Pagination cog

### LINKED ISSUE #352: pagination does not work when clicking LAST_EMOJI
## Description
Right now, when using the pagination, clicking the LAST_EMOJI will raise an exception

```py
  File "/bot/bot/pagination.py", line 392, in paginate
    current_page = len(paginator.pages - 1)
TypeError: unsupported operand type(s) for -: 'list' and 'int'
```

## Proposed Implementation
This is because `paginator.pages` is a list. I believe it should be
```py
current_page = len(paginator.pages) - 1
```

This line of code can be found at [bot/paginator.py](https://github.com/python-discord/seasonalbot/blob/7ef23b4d42716883dd8d51d0b2ffab36e45b6f59/bot/pagination.py#L392)

### PR DESCRIPTION
**Pull request by F4zii**
When using pagination in the Reddit cog, clicking the LAST_EMOJI would
raise an exception like this:
TypeError: unsupported operand type(s) for -: 'list' and 'int'

This was resolved by taking the subtraction out of the len() function.

## Additional Changes
Swapping to the trash can emoji with this PR to align with the changes made to the bot.
https://github.com/python-discord/bot/pull/625

## Relevant Issues
Closes #352 

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

**Comment by F4zii** [comment]
@sco1 Migrated, waiting for your approval

File: bot/pagination.py [hunk 1]
```diff
@@ -261,8 +261,8 @@ def event_check(reaction_: Reaction, user_: Member) -> bool:
 
                 await message.edit(embed=embed)
 
-        log.debug("Ending pagination and removing all reactions...")
-        await message.clear_reactions()
+        log.debug("Ending pagination and deleting the message")
+        await message.delete()
```
Comment (ikuyarihS) [inline bot/pagination.py]: When `asyncio.TimeoutError` happened ( No more reaction from user in set amount of time ) then the default behavior will be to keep the result but remove all the reactions, if you change it to delete, the result from the query will also be gone.

In this case, we want to keep the message and only clear all of its reactions.

File: bot/pagination.py [hunk 2]
```diff
@@ -424,5 +424,5 @@ def check_event(reaction_: Reaction, member: Member) -> bool:
 
             await message.edit(embed=embed)
 
-        log.debug("Ending pagination and removing all reactions...")
-        await message.clear_reactions()
+        log.debug("Ending pagination and deleting the message")
+        await message.delete()
```
Comment (ikuyarihS) [inline bot/pagination.py]: Same with the above, we want to keep the message and only clear the reactions.

File: bot/pagination.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 LEFT_EMOJI = "\u2B05"    # [:arrow_left:]
 RIGHT_EMOJI = "\u27A1"   # [:arrow_right:]
 LAST_EMOJI = "\u23ED"    # [:track_next:]
-DELETE_EMOJI = "\u274c"  # [:x:]
+DELETE_EMOJI = "<:trashcan:637136429717389331>"  # [:trashcan:]
 
 PAGINATION_EMOJI = [FIRST_EMOJI, LEFT_EMOJI, RIGHT_EMOJI, LAST_EMOJI, DELETE_EMOJI]
```
Comment (ikuyarihS) [inline bot/pagination.py]: A minor changes & considerations. Otherwise the functionality works well.
This can be changed from a list to a tuple instead.

Comment (F4zii) [reply bot/pagination.py]: Sure, will do. I just saw that this is the implementation at https://github.com/python-discord/bot
Should I also apply this migration in the /python-discord/bot?

Comment (ikuyarihS) [reply bot/pagination.py]: Should, yes, but doing a whole PR just to change a line, I don't think it's worth it, unless you want to refactor it, then yes, please do!

Comment (F4zii) [reply bot/pagination.py]: Done, waiting for your approval on both bot repos 👍

**Comment by F4zii** [comment]
@sco1 Is everything ready or there are still changes needed?

File: bot/pagination.py [hunk 4]
```diff
@@ -10,9 +10,9 @@
 LEFT_EMOJI = "\u2B05"    # [:arrow_left:]
 RIGHT_EMOJI = "\u27A1"   # [:arrow_right:]
 LAST_EMOJI = "\u23ED"    # [:track_next:]
-DELETE_EMOJI = "\u274c"  # [:x:]
+DELETE_EMOJI = "<:trashcan:637136429717389331>"  # [:trashcan:]
```
Comment (ikuyarihS) [inline bot/pagination.py]: The PR is good to be merged, but before that, let's move this to `constants.py` so we can easily change it later and will make sure that the trashcan used elsewhere will be the same as this one.

Comment (F4zii) [reply bot/pagination.py]: Yeah, as I said, I will add it to the constants module.
Will it be a good idea to try to clear things out and try to solve #348?

**Pull request merged**